ces_client/utils.py

```diff
@@ -65,4 +68,20 @@ def make_answers(self, state):
         answers.append({'text': 'Goodbye!',
                         'value': settings.DECISIONTREE_SESSION_END_TRIGGER})
 
-        return answers
\ No newline at end of file
+        return answers
+
+    @classmethod
```

Comment:
you don't actually ever call this except on an instance. (except for the test). Why do you want this to be a class method?

Reply:
yes, I actually made it a class method, so my tests would be _slightly_ cleaner (i.e., I can access this method without instantiating a new object in the tests). I actually do not have any other reason to scope it as such....which led me to think about "when to use @classmethod generally"?

Reply:
this seems like a good place to use a @staticmethod

Within the larger python world, @classmethods are not very common. They are more common within the Django world, where they are often used to do queries.

DjangModel.objects.all() is working on a class not an instance. We can talk more offline about this too.

Reply:
`DjangModel.objects.all()` - oh, of course. That's a very clear use case. 

yes, it might be nice to talk about these various decorators, especially since the internet has lots of opinions ([this being one of them](https://www.webucator.com/blog/2016/05/when-to-use-static-methods-in-python-never/)).

any way....I marked it as a static method.